src/Estimators/EnergyDensityInput.h

```diff
@@ -0,0 +1,89 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2023 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// Some code refactored from: QMCHamiltonian/EnergyDensityEstimator.h
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_ENERGY_DENSITY_INPUT_H
+#define QMCPLUSPLUS_ENERGY_DENSITY_INPUT_H
+
+#include "InputSection.h"
+#include "ReferencePointsInput.h"
+#include "SpaceGridInput.h"
+#include "type_traits/template_types.hpp"
+
+namespace qmcplusplus
+{
+namespace testing
+{
+template<typename T>
+class EnergyDensityTests;
+}
```

Comment:
Why do we need these lines? Seems not used.

Reply:
Seems not for this PR. I removed it.